rpc-cli/main.go

```diff
@@ -19,8 +21,12 @@ func main() {
 	}
 
 	u, method, body := argv[0], argv[1], argv[2]
+	if uo, err := url.Parse(u); err == nil {
+		uo.Host = srvclient.MaybeSRV(uo.Host)
+		u = uo.String()
+	}
 	var ret interface{}
-	err := rpcutil.JSONRPC2RawCall(u, &ret, method, body)
+	err := rpcutil.JSONRPC2RawCall(srvclient.MaybeSRV(u), &ret, method, body)
```

Comment:
what would happen if you passed in a url into `MaybeSRV` like you are here? I feel like it would do something weird